### PR TITLE
PodsDFVText

### LINKED ISSUE #4662: DFV: Text field type
DFV: Text field type 

- [x] Placeholder
- [x] Default value 
- [x] Max length
- [x] Readonly
- [x] Required
- [x] Additional CSS Classes

### PR DESCRIPTION
**Pull request by pglewis**
Fixes #4662

### CODE REVIEW

**Comment by pglewis** [comment]
Also @sc0ttkclark, I believe we can delete the old PHP version in `ui/fields`?

File: ui/js/pods-dfv/_src/text/text.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class PodsDFVText extends React.Component {
+	constructor ( props ) {
```
Comment (sc0ttkclark) [inline ui/js/pods-dfv/_src/text/text.js]: Could this type of stuff be put in a new PodsDFVReactField type of class where it does the most common stuff and you just extend it to customize?

**Comment by pglewis** [comment]
Definitely makes sense to at least start off by encapsulating the propTypes that will be used everywhere, so done.  Ready for review again.

**Comment by pglewis** [comment]
With password: 
- [ ] Remove prototyping code in `password::input` put there to switch between DFV and the old field
- [ ] Remove old PHP password field
- [ ] Sticking `fieldType` directly on the input's type attrib isn't a great generic solution, create a mapping function

**Pull request merged**